go-selinux/selinux_stub_test.go

```diff
@@ -22,6 +22,13 @@ func TestSELinuxStubs(t *testing.T) {
 		t.Error(err)
 	}
 
+	if _, err := LfileLabel("/etc"); err != nil {
+		t.Error(err)
+	}
+	if err := LsetFileLabel("/etc", testLabel); err != nil {
```

Comment:
This could be dangerous if run as root.

Reply:
Yes, I was looking for a good path to use (this is for the stubs, so _should_ not do anything, but the existing tests also used `/etc`)

Reply:
Updated to use a temp-file to test with 👍

Reply:
If we are using system directories elsewhere we probably should not.